modules/microshift-install-rhel-image-mode-csi-vgloop.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-csi_{context}"]
+= Configuring container storage for the bootc image and the {microshift-short} CSI
+
+Use this procedure when a volume group with the necessary storage is not available on the host. Create and allocate an additional LVM volume group loop device to be shared with the container and used by the {microshift-short} Logical Volume Manager (LVM) Container Storage Interface (CSI).
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have `sudo` permissions.
+
+.Procedure
+
+. Create a file to be used for LVM partitioning by running the following command:
++
+[source,terminal]
+----
+VGFILE=/var/lib/microshift-lvm-storage.img
+
+sudo losetup -f "${VGFILE}"
+----
+
+. Resize the file to your specified size by running the following command:
++
+[source,terminal]
+----
+VGSIZE=1G # <1>
+
+$ sudo truncate --size="${VGSIZE}" "${VGFILE}"
+----
+<1> Specify the size for the file.
+
+. Query the loop device name by running the following command:
++
+[source,terminal]
+----
+$ VGLOOP=$(losetup -j ${VGFILE} | cut -d: -f1)
+----
+
+. Create a free volume group on the loop device by running the following command:
++
+[source,terminal]
+----
+$ sudo vgcreate -f -y rhel "${VGLOOP}"
```

Comment:
Not sure if you want to include this, but a user might not know the name of their existing volume group.

`sudo vgs` is the command to discover that. A user wants to use a name different than the existing name. In this case `rhel`.

Reply:
I thought we are creating the volume group here, not using an existing one?

Reply:
There might be an existing volume group that was created during install.  The volume group `rhel` is mentioned in this document
https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/4.16/html/installing/microshift-install-rpm#microshift-install-rpm-preparing_microshift-install-rpm
Probably just a small nit on my part.

Reply:
Right, but we checked for one in the preceeding procedure here, https://github.com/openshift/openshift-docs/pull/80599/files#diff-27c086846240b4b4c93bfcb025deacfb945413c72205b6c07161fe41b9dc7cdeR22 

The current procedure is for when we did not find one. See https://github.com/openshift/openshift-docs/pull/80599/files#diff-27c086846240b4b4c93bfcb025deacfb945413c72205b6c07161fe41b9dc7cdeR40

Reply:
ah, I see it now. Please close the comment. Nothing to do here,